src/components/Code-search/CodeSearchCard.tsx

```diff
@@ -14,16 +13,11 @@ export const CodeSearchCard: FunctionComponent = () => (
     >
         <div className="sg-bg-code-search-new-cta hover:cta-free-cody relative overflow-hidden rounded-2xl border border-gray-200 border-opacity-25 !px-14 py-16 md:w-1/2 md:p-16">
             <div className="bg-grad absolute right-0 top-0 h-[3px] w-full flex-1 bg-gradient-to-r from-blue-300 via-violet-400 to-vermillion-300" />
-            <Heading size="h2" className="text-5xl !leading-10 !tracking-[-1px] text-white">
-                Code Search Enterprise
-            </Heading>
-            <Heading
-                size="h3"
-                className="py-4 !text-[18px] !leading-[27px] !tracking-[-0.25px] text-white text-opacity-80"
-            >
+            <h2 className="text-white">Code Search Enterprise</h2>
+            <h3 className="py-4 text-white text-opacity-80">
                 Get Code Search for your team’s entire private codebase. Contact us for a demo or to get started with a
                 free trial.
-            </Heading>
+            </h3>
             <div className="flex max-w-[356px] flex-col flex-wrap gap-4 pt-4 sm:flex-row">
                 <Link href="/demo" className="btn btn-primary max-w-[216px]">
```

Comment:
There is something weird happening with this button 
<img width="664" alt="Screenshot 2024-06-19 at 11 33 41 AM" src="https://github.com/sourcegraph/about/assets/102541294/ef9c4ae7-de77-4b45-aa8e-fb1eaf400e57">